tests/DoctrineTest/InstantiatorTest/Exception/InvalidArgumentExceptionTest.php

```diff
@@ -34,7 +34,7 @@ class InvalidArgumentExceptionTest extends PHPUnit_Framework_TestCase
 {
     public function testFromNonExistingTypeWithNonExistingClass()
     {
-        $className = __CLASS__ . uniqid();
+        $className = __CLASS__ . uniqid('', true);
```

Comment:
Yes, but the class name is invalid. Can you add a `str_replace` around it, please?